chaudloader/src/hooks/stage0.rs

```diff
@@ -225,8 +225,9 @@ fn init(
         if on_game_load_hook_needed {
             super::stage1::install_on_game_load(&game_env)?;
         }
-        if pck_load_hook_needed {
+        if audio_hooks_needed {
             super::stage1::install_pck_load(&game_env)?;
+            super::stage1::install_bnk_load(&game_env)?;
```

Comment:
Is there a reason why we would always want to install both pck load and bnk load together? If not I would suggest to keep them separate

Reply:
Well actually my original reason for adding these kings of checks before adding the hooks was to avoid writing to the game executable unless it was really necessary.  This was to prevent triggering any past or future anti tamper checks.
If one is being written it would already trigger anti tampers, in theory this could just be a single `exe_hooks_needed` check for all 3 hooks?